spec/support/xml_helper.rb

```diff
@@ -21,4 +21,13 @@ def delete_xml_at_xpath(xml, xpath)
     element.parent.delete(element)
     document.to_s
   end
+
+  def pretty_xml_from_string(document)
+    output = String.new(encoding: 'UTF-8')
+    doc = REXML::Document.new(document)
+    formatter = REXML::Formatters::Pretty.new
+    formatter.compact = true
```

Comment:
It is not super significant which formatter we use, but Pretty generates nicely-formatted XML that's actually fairly close to what we have.

`compact = true` keeps `<foo>bar</foo>` from becoming:

```xml
<foo>
  bar
</foo>
```